### PR TITLE
add instruction for pkcs 12 update

### PR DESCRIPTION
**Pull request by zhang-shengping**

### CODE REVIEW

File: docs/p12-limitation.rst [hunk 1]
```diff
@@ -3,5 +3,21 @@
 Configure terminated HTTPS load balancer with PKCS12 certificate bundle
 =======================================================================
 
-Overview
---------
+F5 Neutron Lbaas dashboard does not support load balancer SSL offloading function with PKCS 12 format certificates.
```
Comment (zongzw) [inline docs/p12-limitation.rst]: dashboard -> Dashboard?

File: docs/p12-limitation.rst [hunk 2]
```diff
@@ -3,5 +3,21 @@
...
+ # extract certificate with encrypt private keys
+ openssl pkcs12 -in certificate.pfx -out certificate.txt
+
+ # extract certificate without encrypt private key
+ openssl pkcs12 -in certificate.pfx -out certificate.txt -nodes
+
+- Make sure the extracted certificates are X509 format.
+- Copy corresponding certificates and private key from extracted file to F5 Neutron Lbaas Dashboard.
+- If the PKCS 12 bundle is extracted with encrypt private key, the new passphrase also needs to be provide to F5 Neutron Lbaas Dashboard.
+- Click ``create`` or ``update`` on the Dashboard to create or update SSL offloading for a load balancer.
```
Comment (zongzw) [inline docs/p12-limitation.rst]: So we should make it clear where's the ``create`` or ``update``,  
SSL offloading for a load balancer.. for a listener??

File: docs/p12-limitation.rst [hunk 3]
```diff
@@ -3,5 +3,21 @@
...
+
+.. code-block:: bash
+
+ # extract certificate with encrypt private keys
+ openssl pkcs12 -in certificate.pfx -out certificate.txt
+
+ # extract certificate without encrypt private key
+ openssl pkcs12 -in certificate.pfx -out certificate.txt -nodes
+
+- Make sure the extracted certificates are X509 format.
```
Comment (zongzw) [inline docs/p12-limitation.rst]: 'make sure' do you mean 'verify'?

File: docs/p12-limitation.rst [hunk 4]
```diff
@@ -3,5 +3,21 @@
 Configure terminated HTTPS load balancer with PKCS12 certificate bundle
 =======================================================================
 
-Overview
---------
+F5 Neutron Lbaas Dashboard does not support load balancer SSL offloading function with PKCS 12 format certificates.
+
+The PKCS 12 bundle must be extracted before using F5 Neutron Lbaas Dashboard.
+
+- To extract a PKCS 12 bundle use openssl tool.
```
Comment (zongzw) [inline docs/p12-limitation.rst]: - Extract a PKCS 12 bundle via openssl tool.
??

**Review by zongzw — CHANGES_REQUESTED** [review]

File: docs/p12-limitation.rst (hunk 3 above)
Comment (zhang-shengping) [reply docs/p12-limitation.rst]: The user does not have to verify it to make sure the X509 format.

Comment (zongzw) [reply docs/p12-limitation.rst]: I mean how does user to 'make sure'?

**Review by zongzw — APPROVED** [review]
Looks great  to me.

**Review by zhaoqin-github — CHANGES_REQUESTED** [review]
Suggest to:

a) use "-clcerts -nokeys" option to extract server cert.

b) use "-cacerts -nokeys" option to extract intermediate certs

c) use "-nocerts" option to extract the private key

d) the extracted intermediated certs need to be packaged into pkcs7 file, before importing to Barbican.

File: docs/p12-limitation.rst [hunk 5]
```diff
@@ -2,6 +2,276 @@
 
 Configure terminated HTTPS load balancer with PKCS12 certificate bundle
 =======================================================================
+F5 Neutron Lbaas Dashboard only support SSL offloading with PEM and PKCS 7 certificate.
```
Comment (zhaoqin-github) [inline docs/p12-limitation.rst]: Maybe we can change it to "... with PEM formatted certificate."

**Review by zhaoqin-github — CHANGES_REQUESTED** [review]

**Review by zhaoqin-github — APPROVED** [review]
Approve. We may update the document further according to future testing result.

**Pull request merged**